### ISSUE TITLE
Option to automatically wrap all names in square brackets

### ISSUE THREAD

**Issue by mezlondr** [dbeaver/dbeaver#24122]
**Is your feature request related to a problem? Please describe.**
Currently, (afaik) there is not a way to automatically wrap names with square brackets. Even when the square brackets are typed manually, the autocomplete removes them.

**Describe the solution you'd like**
For example, typing something like
`select * from Use`

after selecting the suggestion, with this option enabled should autocomplete to 
`select * from [Users] [u] `

the same goes for all column names, procedure names, etc., just everything that is some kind of a name:

`exec [dbo].[SomeProcedure]`

**Describe alternatives you've considered**
Typing it manually

**Additional context**
It is a convention to wrap all names in square brackets so that searching is easier (without false positives).

**Comment by E1izabeth** [comment]
Similar to #23020

**Comment by mezlondr** [comment]
Yes, although this focuses on all SQL editors (as a global convention), not just generating.
So the users could have options to pick their convention (grave accent/square brackets/are there any other?)

With the option I am suggesting enabled, **all** names would be wrapped according to the set convention (without the option enabled perhaps only **reserved** words could be escaped automatically)

**Comment by E1izabeth** [comment]
I see. Thank you for detailed explanation
